site/app/controllers/admin/ReportController.php

```diff
@@ -138,10 +138,6 @@ public function getGradeSummariesLastRun() {
      */
     #[Route("/courses/{_semester}/{_course}/reports/csv")]
     public function generateCSVReport() {
-        if (!$this->core->getUser()->accessAdmin()) {
-            $this->core->getOutput()->showError("This account cannot access admin pages");
-        }
-
```

Comment:
While this is a good change to make, it doesn't seem to be related to the rest of this PR, and should be done separately.